src/easydmp/dmpt/admin.py

```diff
@@ -173,7 +173,7 @@ class SectionAdmin(ObjectPermissionModelAdmin):
             'fields': ('super_section', 'section_depth'),
         }),
         ('Advanced', {
-            'fields': ('cloned_from', 'cloned_when'),
+            'fields': ('cloned_from', 'cloned_when', 'optional', 'optional_text'),
```

Comment:
I still don't think optional_text is needed.

Reply:
There could be a help-text to "optional": "Remember to flesh outthe BooleanQWuestion in position 0!" or some such.